flow/raster_cache.cc

```diff
@@ -40,76 +43,31 @@ void RasterCacheResult::draw(SkCanvas& canvas, const SkPaint* paint) const {
 }
 
 RasterCache::RasterCache(size_t access_threshold,
-                         size_t display_list_cache_limit_per_frame)
+                         size_t picture_and_display_list_cache_limit_per_frame)
```

Comment:
the `picture_and_` part can be removed now, as well as any methods related to caching pictures - that functionality has been removed from the engine.